pom.xml

```diff
@@ -175,6 +191,95 @@
       </pluginManagement>
 
         <plugins>
+            <plugin>
+              <groupId>org.openapitools</groupId>
+              <artifactId>openapi-generator-maven-plugin</artifactId>
+              <version>${openapi-generator.version}</version>
+              <executions>
+                <execution>
+                  <id>generate-api-and-models</id>
+                  <phase>generate-sources</phase>
+                  <goals>
+                    <goal>generate</goal>
+                  </goals>
+                  <configuration>
+                    <inputSpec>
+                      ${project.basedir}/src/main/resources/META-INF/openapi.yml
+                    </inputSpec>
+                    <output>${project.build.directory}/generated-sources/openapi-spec</output>
+
+                    <skipValidateSpec>true</skipValidateSpec>
+
+                    <skipIfSpecIsUnchanged>false</skipIfSpecIsUnchanged>
+                    <cleanupOutput>false</cleanupOutput>
+
+                    <generatorName>jaxrs-spec</generatorName>
```

Comment:
Do you mind adding a comment to the top of this plugin to "https://openapi-generator.tech/docs/generators/jaxrs-spec" just so folks can look up what all these flags mean incase they are interested/need to make a change?

Reply:
Yes, good idea. I'll add that.